plugins/modules/swim_intent.py

```diff
@@ -741,8 +753,8 @@ def get_have(self):
 
             elif self.have.get("imported_image_id"):
                 have["activation_image_id"] = self.have.get("imported_image_id")
-
             else:
+                self.log("Image details for activation not provided", "CRITICAL")
```

Comment:
self.log("Image details required for activation have not been provided", "CRITICAL")

Reply:
updated.